add support for Fossil Gen 6

1337Misom opened the pull request:
Add support for Fossil Gen 6 (fossil-hoki), a smartwatch with sdm429w chipset.

barni2000 commented on sdm429-fossil-hoki.dts:
```diff
@@ -0,0 +1,7 @@
+/dts-v1/;
+
+/ {
+    qcom,board-id = <0x10b 0x0a>;
```
```suggestion
	qcom,board-id = <0x10b 0x0a>;
```

1337Misom replied:
Fixed

1337Misom committed:
add support for Fossil Gen 6

Add support for Fossil Gen 6 (fossil-hoki), a smartwatch with sdm429w chipset.

Signed-off-by: Misom <misom@symlinks.dev>
